rmf_task_ros2/include/rmf_task_ros2/bidding/Response.hpp

```diff
@@ -22,6 +22,7 @@
 #include <rmf_task_msgs/msg/bid_notice.hpp>
 #include <rmf_task_msgs/msg/bid_proposal.hpp>
 #include <rmf_task_msgs/msg/bid_response.hpp>
+#include <optional>
```

Comment:
Let's leave this unrelated change for a separate PR.

Reply:
Hi, just wanted to add some details for this addition:

Some of our computers were having problems building without this include line.
This was the error that would show up: `'optional' in namspace 'std' does not name a template type`

after the addition of the include it built fine!

Reply:
I see, thanks for clarifying.